src/RileyMaze/userFunction.js

```diff
@@ -240,6 +232,22 @@ ${code}
             return;
         }
 
+        if (scope.hasOwnProperty('riley') && scope.riley.queue.length !== MAX_QUEUE_LEN) {
+            // Add lines for the function name
+            let lineNum = MAX_QUEUE_LEN + 2;
```

Comment:
I don't feel very comfortable with guessing the line number, as it can easily lead to unintentionally confusing error messages when the player does something we don't expect:
![Screenshot from 2019-04-26 12-22-46](https://user-images.githubusercontent.com/459371/56831681-8fa87400-681e-11e9-990b-6b2bde00444b.png)
If I'm reading the code correctly, you removed the error-throwing code from the methods of the Riley object because of https://phabricator.endlessm.com/T26436 ? I think I would prefer here to reinstate those, and instead fix the underlying bug in a separate PR. Throwing the error seems to be the correct way to get the error message at the correct line, but it just happens to not work at the moment...

Reply:
Yeah, I agree, thanks!